grunt/index.d.ts

```diff
@@ -796,8 +796,8 @@ declare namespace grunt {
              *
              * @note taskFunction.apply(scope: grunt.task.IMultiTask<any>, args: any[])
              */
-            registerMultiTask(taskName: string, taskFunction: Function): void
-            registerMultiTask(taskName: string, taskDescription: string, taskFunction: Function): void
+            registerMultiTask<T extends any>(taskName: string, taskFunction: (this: IMultiTask<T>, ...args: any[]) => void): void;
+            registerMultiTask<T extends any>(taskName: string, taskDescription: string, taskFunction: (this: IMultiTask<T>, ...args: any[]) => void): void;
```

Comment:
I don't think I understand what you mean with unchecked?

>Can't the caller pass registerMultiTask<any>(...) or registerMultiTask<MyOptions with any>(...)?

That was my intention? 

Actually `T extends any` does not default `T` to `any`, it's just like having just `T` it defaults to `{}`. Maybe just change this to `IMultiTask<any>`?